driver-core/src/main/java/com/datastax/driver/core/AggregateMetadata.java

```diff
@@ -101,11 +106,29 @@ static AggregateMetadata build(KeyspaceMetadata ksm, Row row, VersionNumber vers
         if (version.getMajor() >= 3) {
             stateType = DataTypeCqlNameParser.parse(stateTypeName, cluster, ksm.getName(), ksm.userTypes, null, false, false);
             String rawInitCond = row.getString("initcond");
-            initCond = rawInitCond == null ? null : codecRegistry.codecFor(stateType).parse(rawInitCond);
+            if (rawInitCond == null) {
+                initCond = null;
+            } else {
+                try {
+                    initCond = codecRegistry.codecFor(stateType).parse(rawInitCond);
```

Comment:
I've tested with 3.0.4 and 3.4 which work.  < 3.4 and 3.0.4 raise an exception because of [CASSANDRA-11064](https://issues.apache.org/jira/browse/CASSANDRA-11064), so the code isn't really exercised.  

One concern I have is that the aggregate cql string we generate (`CREATE AGGREGATE ks_1.ag() SFUNC id STYPE 'org.apache.cassandra.db.marshal.DynamicCompositeType(s=>org.apache.cassandra.db.marshal.UTF8Type,i=>org.apache.cassandra.db.marshal.Int32Type)' INITCOND 0x80730003666f6f00806900040000002000;`) does work if we execute it against 3.0.4 and 3.4, however it fails against 2.2.5 with the error:

>  Invalid component representation: 0x80730003666f6f00806900040000002000)